contracts/interfaces/IRamm.sol

```diff
@@ -82,12 +82,15 @@ interface IRamm {
 
   function removeBudget() external;
 
+  function setEmergencySwapPause(bool _swapPaused) external;
```

Comment:
`NXMaster.setEmergencyPause` is **public**

but I set `setEmergencySwapPause` as **external** because Ramm does not call it internally - just want to check this is what we want